commune.lic

```diff
@@ -0,0 +1,412 @@
+	            'meraud' => 'wolf', 'firulf' => 'owl', 'kerenhappuch' => 'shrew',
+	            'damaris' => 'panther', 'phelim' => 'nightingale', 'dergati' => 'shrike',
+	            'everild' => 'boar', 'kuniyo' => 'wolverine', 'trothfang' => 'centaur',
+	            'truffenyi' => 'ox', 'alamhif' => 'magpie', 'huldah' => 'weasel',
+	            "hav'roth" => 'cobra', "peri'el" => 'kingsnake', 'ushnish' => 'viper',
+	            'eluned' => 'dolphin', 'lemicus' => 'albatross', 'drogor' => 'shark',
+	            'glythtide' => 'ram', 'saemaus' => 'donkey', "be'ort" => 'coyote',
+	            'tamsine' => 'cat', 'albreda' => 'dove', 'harawep' => 'spider',
+	            'faenella' => 'wren', 'murrula' => 'phoenix', 'idon' => 'heron',
+	            'chadatru' => 'lion', 'rutilor' => 'mongoose', 'botolf' => 'goshawk',
+	            "urrem'tier" => 'scorpion', 'eylhaar' => 'jackal', 'aldauth' => 'vulture'
+		      }
+	if symbols.key?("#{bead_type}")
+	  @shape = symbols["#{bead_type}"]
+	elsif symbols.value?("#{bead_type}")
+	  @shape = bead_type
+	else
+	  missing_item('a valid carving target')
+	end
+	echo "Carving: #{@shape}"
+    if GameObj.right_hand.noun == 'block'
+	  carve_bead("#{@shape}")
+	elsif exists?('block')
+	  carve_bead("#{@shape}")
+	else
+	  carve_block
+	  carve_bead("#{@shape}")
+	end
+  end
+ 
+  def carve_block()
+    if have_cleric_items?(['holy water']) && exists?('carving knife') then
+	  empty_cleric_hands
+	  if forage?('limb')
+	    prepare?('bless', 1)
+	    sprinkle_holy_water('my limb')
+        bput('cast my limb', 'You gesture at', "You don't have")
+        bput('get my carving knife', 'You get', "I don't know", "What were you")
+		if GameObj.right_hand.noun == 'limb' && GameObj.left_hand.noun == 'knife' then
+		  while GameObj.right_hand.noun == 'limb'
+		    bput('carve my limb with my carving knife', 'You begin to hack', 'With fluid strokes', 'You continue to whittle', 'You grow more careful', 'With a final deep cut')
+		  end
+		else
+		  missing_item('limb or carving knife')
+		end
+		waitrt
+		empty_cleric_left_hand
+	  else
+		missing_item('a limb')
+	  end
+	end
+  end
+  
+  def carve_bead(shape)
+    if exists?('shaper')
+	  empty_cleric_left_hand if GameObj.left_hand.noun != 'shaper'
+	  empty_cleric_right_hand if GameObj.right_hand.noun != 'block'
+	  fput('get my block') if GameObj.right_hand.noun != 'block'
+	  fput('get my shaper') if GameObj.left_hand.noun != 'shaper'
+	  while GameObj.right_hand.noun == 'block'
+	    case bput("shape my block to #{shape}", 'You fumble around but', 'Roundtime') # You skillfully begin', 'You adeptly begin', 'You carefully begin shaping', 'You confidently begin', 'Trying your best', 'With heavy strokes', 'You focus carefully', 'You focus slowly', 'You focus clumsily', 'You smile with satisfaction', 'You notice the', 'Pursing your lips', 'Your spirit soars with pride', 'You fumble around but'
```

Comment:
I was adding them as I tested but got new messages every test run. I found that using 'Roundtime' worked effectively without using each individual message. I will remove them.